plugins/module_utils/network/ios/facts/bgp_address_family/bgp_address_family.py

```diff
@@ -142,7 +144,9 @@ def populate_facts(self, connection, ansible_facts, data=None):
             )
 
             params = utils.remove_empties(
-                utils.validate_config(self.argument_spec, {"config": objs})
+                bgp_af_parser.validate_config(
```

Comment:
@NilashishC are there any more changes expected to fix the no-log issue from RM side as sanity docker tests for devel and 2.11 with same no-log failures?